proto/logtail.proto

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 Matrix Origin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+package logtail;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "timestamp.proto";
+import "api.proto";
+
+option go_package = "github.com/matrixorigin/matrixone/pkg/pb/logtail";
+option (gogoproto.sizer_all)      = false;
+option (gogoproto.protosizer_all) = true;
+
+// subscribe table request
+message SubscribeRequest {
+    timestamp.Timestamp cn_want = 1;
```

Comment:
What is the meaning of this field?

Reply:
https://github.com/matrixorigin/matrixone/blob/a39641fb659a2c72721e53a2b66f77cc38b3fac3/pkg/pb/api/api.pb.go#L297
It's similar with `api.SyncLogTailReq.CnWant`.
Do you think that `SubscribeRequest.CnWant` isn't nesessary?

Reply:
https://github.com/matrixorigin/matrixone/blob/a39641fb659a2c72721e53a2b66f77cc38b3fac3/pkg/pb/api/api.pb.go#L297
It's similar with `api.SyncLogTailReq.CnWant`.
Do you think that `SubscribeRequest.CnWant` isn't necessary?

Reply:
`cn_want` had been removed in the following commit.